test/rule-matches/frame-focusable-content-matches.js

```diff
@@ -0,0 +1,57 @@
+describe('frame-focusable-content-matches', function() {
+  'use strict';
+  var rule;
+
+  beforeEach(function() {
+    rule = axe.utils.getRule('frame-focusable-content');
+  });
+
+  it('returns false for the top-level context', function() {
+    var result = rule.matches(null, null, { initiator: true });
```

Comment:
This test doesn't prove what you say it does. It could also be false because there's no with/height. I think we need to make sure all props exist in these tests.